### PR TITLE
fixes spacetime distortions moving through spacetime

### PR DESCRIPTION
**Pull request by PKPenguin321**
by anchoring them

https://www.youtube.com/watch?v=SM69WsJ8r9k

Fixes https://github.com/tgstation/tgstation/issues/18510

### CODE REVIEW

**Comment by PKPenguin321** [comment]
there was 0 testing and very little thought put into this so @xxalpha if you could double check that anchoring these won't break anything that would be nice of you

**Comment by MrStonedOne** [comment]
it shouldn't, this is exactly what anchoring is designed for.

**Comment by xxalpha** [comment]
Why aren't effects immune to these things anyways.

**Comment by xxalpha** [comment]
Why aren't effects anchored by default

**Comment by optimumtact** [comment]
Because we have no idea what may or may not rely on that behaviour

File: code/modules/spells/spell_types/spacetime_distortion.dm [hunk 1]
```diff
@@ -62,6 +62,7 @@
 	desc = "A distortion in spacetime. You can hear faint music..."
 	icon_state = "wave1"
 	color = "#8A2BE2"
+	anchored = 1
```
Comment (xxalpha) [inline code/modules/spells/spell_types/spacetime_distortion.dm]: @PKPenguin321 if it's not a huge bother to make another commit, define this ``anchored = 1`` in the parent type ``/obj/effect/cross_action``
